### PR TITLE
Adding "kind" to the expected response.

### PR DESCRIPTION
**Pull request by JannikZed**
I want to add this highly relevant config variable, that we needed to create our custom pre-payment payment gateway.

### CODE REVIEW

File: docs/developer/extending/apps/synchronous-webhooks.mdx [hunk 1]
```diff
@@ -164,6 +164,7 @@ Below is the list of fields that webhook can return in the JSON response for a p
 
 - `action_required` - (boolean) determines if additional action (confirmation) is required.
 - `action_required_data` - (object) additional data that may be needed to confirm the payment, for example, URL to redirect the user to the confirmation page in the frontend.
+- `kind` - (string) Transaction kind, one of: action_to_confirm, auth, cancel, capture, capture_failed, confirm, external, pending, refund, refund_failed, refund_ongoing, refund_reversed, void.
```
Comment (korycins) [inline docs/developer/extending/apps/synchronous-webhooks.mdx:167]: Hey @JannikZed,
Thanks for pointing out this!
Can you also update below example to include `kind` field? 🙏 
https://github.com/saleor/saleor-docs/pull/467/files#diff-0358f49f3174a451a396b8a613ae5382495a6bb57b9c7a8b9d6b8af999a9f425R184

Comment (JannikZed) [reply docs/developer/extending/apps/synchronous-webhooks.mdx:167]: @korycins I added a new example on how to use the "kind" field.

Comment (korycins) [reply docs/developer/extending/apps/synchronous-webhooks.mdx:167]: Thanks!

**Review by korycins — APPROVED** [review]

**Comment by peelar** [comment]
Thanks for your contribution @JannikZed, you are a star ⭐!

**Review by peelar — APPROVED** [review]

**Comment by peelar** [comment]
Ah, I just noticed something.

Since you submitted your PR @JannikZed, we split up the synchronous webhooks documentation. The changes you've made were in `docs/developer/extending/apps/synchronous-webhooks.mdx`, but now the path should be `docs/developer/extending/apps/synchronous-webhooks/payment-webhooks.mdx`. 

If you want to remain the author of these changes, unfortunately, you are forced to rebase the branch and move your changes to the correct file. I can handle that for you but then I will be the author.

I don't want to steal your spotlight without your permission @JannikZed 😅, so please let me know how you want to handle it.

**Comment by peelar** [comment]
> @peelar you can do that - no problem :)

I ended up creating a new PR, if you don't mind. It's available here: https://github.com/saleor/saleor-docs/pull/488

**Pull request closed without merging**